tests/components/google/test_calendar.py

```diff
@@ -1301,6 +1301,7 @@ async def test_event_differs_timezone(
     }
 
 
+@pytest.mark.freeze_time("2023-11-30 12:15:00 +00:00")
```

Comment:
How was the datetime selected? It doesn't match any of the recurrence dates, which seem to be one Friday a month. Is that correct?

Reply:
It overlaps with one of the Fridays. Is that the idea?

Reply:
Ok, I guess we can merge for now if it passes, but @allenporter should follow up and check if this is intended.